ddtrace/logger.py

```diff
@@ -0,0 +1,40 @@
+import logging
+from logging.handlers import RotatingFileHandler
+import os
+
+from ddtrace.internal.utils.formats import asbool
+
+
+DEFAULT_FILE_SIZE_BYTES = 15 << 20  # 15 MB
+
+
+def configure_ddtrace_logger():
+    # type: () -> None
+    """ """
```

Comment:
TODO: docstring

Reply:
The latest version has a description; is anything else still needed for the `docstring` todo?

Reply:
Marking as resolved.